novels/novel_storage/handler.py

```diff
@@ -0,0 +1,61 @@
+import os
+import json
+from pathlib import Path
+
+
+ROOT_DIRECTORY = Path(__file__).parent.parent
+
+
+def make_novel_directory(novel_key: str) -> None:
+    novel_directory = ROOT_DIRECTORY / novel_key
+    if not novel_directory.exists():
+        os.mkdir(novel_directory)
+    
+    assets = novel_directory / "assets"
+    if not assets.exists():
+        os.mkdir(assets)
+
+    chapters = novel_directory / "chapters.json"
+    if not chapters.exists():
+        with open(chapters, "w") as file:
+            file.write("{}")
```

Comment:
Suggesting to create the file with always a valid structure, respecting the python type with non-optional properties. This will ensure a valid data structure is always returned when reading the file.